README.md

```diff
@@ -176,6 +177,21 @@ Arguments:
   - First: ActiveRecord object, Hash or Array of Hashes;
   - Last: Hash of options (same as `JSONAPI::Utils#jsonapi_render`).
 
+#### Paginators
+
+Paginators can be customized either globally via `JSONAPI.configuration.default_paginator` or per resource. If you are rendering a collection of hashes,
+you'll need to override `Resource#pagination_range`, which returns a range for your resources. For example:
+
+```ruby
+class CustomPaginator < JSONAPI::Paginator
+
+  def pagination_range(page_params)
+    offset = page_params['offset']
+    limit  = JSONAPI.configuration.default_page_size
+    offset..offset + limit - 1
+  end
```

Comment:
@austenito, can you apply just a little change here to make the usage a little bit clearer:

...you'll need to implement the `#pagination_range`method, which returns a range for your resources. For example:
```ruby
class CustomPaginator < JSONAPI::Paginator
  def pagination_range(page_params)
    offset = page_params['offset']
    limit  = JSONAPI.configuration.default_page_size
    offset..offset + limit - 1
  end
```

And then it can be either set at the resource class level (e.g. `UserResource.paginator :custom_paginator`)  or via config initializer:

```ruby
# config/initializers/jsonapi_resources.rb
JSONAPI.configure do |config|
  config.default_paginator = :custom_paginator
end
```
Thanks!